internal/repository/prisma/dbsqlc/workflows.sql

```diff
@@ -364,10 +364,16 @@ INSERT INTO "WorkflowTriggerScheduledRef" (
 ) RETURNING *;
 
 -- name: ListWorkflowsForEvent :many
+WITH LatestWorkflowVersions AS (
```

Comment:
This queries every workflow in the database. It would be more performant to track down the candidate workflow versions first, and then make sure the candidate workflow versions are the latest version.